warehouse/tuf/interfaces.py

```diff
@@ -0,0 +1,53 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from zope.interface import Interface
+
+
+class IKeyService(Interface):
+    def create_service(context, request):
+        """
+        Create the service, given the context and request for which it is being
+        created.
+        """
+
+    def get(rolename, key_type):
+        """Return a key from specific rolename"""
+
+
+class IStorageService(Interface):
+    def create_service(context, request):
+        """
+        Create the service, given the context and request for which it is being
+        created.
+        """
+
+    def get_backend():
```

Comment:
Where is this implemented?

Reply:
I'm doing a clean-up on the part of the interface now.
I'm not familiar as well. This is not required.